python/build_template/configure_build.py

```diff
@@ -0,0 +1,349 @@
+#!/usr/bin/python3
+from lib import ninja_syntax
+from pathlib import Path
+import sys
+import argparse
+import os
+import dx_build
+
+game_name = "Rock Band 3 Deluxe"
+
+ninja = ninja_syntax.Writer(open("build.ninja", "w+"))
+
+#versions
+#gh1, gh2 - " "
+#rb1, track packs - "-v 4"
+#rb2, tbrb, gdrb - "-v 5"
+#rb3, dc, blitz - "-v 6"
+ninja.variable("ark_version", "-v 6")
+ark_version = "-v 6"
+
+#require user provided vanilla ark extract in "vanilla/platform" folder
+vanilla_files = False
+
+#set True for rb1 and newer 
+new_gen = True
+
+#patch for patch arks, main for patchcreator, old gen, and rb1
+hdr_name = "patch"
+
+dtb_encrypt = "-e"
+ark_encrypt = "-e"
+miloVersion = "--miloVersion 26"
+
+#paths in _ark/dx/custom_textures that should generate list dtbs
+custom_texture_paths = [
+    "highways",
+    "streaks",
+    "overdrive",
+    "gems/gems_default",
+    "strikeline/strikeline_guitar",
+    "flames/flames_spark",
+    "sustains",
+    "score/scoreboard_frame",
+    "rails/beat_lines",
+    "stars/score_star_frame",
+    "font",
+    "solo_box",
+    "bre/bre_shield",
+    "rails/rails_track",
+    "lanes/gem_mash_green_emmisive",
+    "overdrive_bar/od_bar_background",
+    "multiplier_ring/multiplier_ring_plate_fc",
+    "crowd_meter/crowd_meter_frame",
+    "keyboard/keyboard_lanes",
+    "vocal_highway/vocal_highway_bg",
+    "vocal_arrows/vocal_arrow",
+    "vocal_note/vocal_note_tube",
+    "vocal_overdrive/vocal_overdrive_now_bar"
+]
+
+#patchcreator options
+patchcreator = False
+new_ark_part = "10"
+dol_name = "main.dol"
+elf_name = " "
+bin_name = " "
+xex_name = " "
+
+parser = argparse.ArgumentParser(prog="configure")
+parser.add_argument("platform")
+parser.add_argument(
+    "--no-updates", action="store_true", help="disable dx song updates"
+)
+
+args = parser.parse_args()
+
+#THIS IS TEMPLATE YOU CAN REMOVE IF YOU DONT NEED IT
+if args.platform == "ps2":
+    #all milo ps2 games from gh to rb use MAIN_0.ARK
+    hdr_name = "MAIN"
+    if new_gen == False:
+        #pre rb2 does not use miloVersion for superfreq image generation on ps2
+        miloVersion = " "
+        #remove these two if rock band 1
+        ark_encrypt = " "
+        dtb_encrypt = dtb_encrypt.upper()
```

Comment:
instead of setting dtb_encrypt to `dtb_encrypt.upper()`, i would directly set it to `-E` so we don't have to dig for the initial value